amethyst_input/src/mouse.rs

```diff
@@ -0,0 +1,21 @@
+use serde::{Deserialize, Serialize};
+
+/// Mouse axis
+#[derive(Eq, PartialEq, Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum MouseAxis {
+    /// The X axis represents moving the mouse left / right
+    X,
+    /// The Y axis represents the mouse moving up / down
+    Y,
+}
+
+/// Mode of how mouse movement is presented to user code
+#[derive(Eq, PartialEq, Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum MouseMode {
+    /// Pass the pixel distance in a normalized way, but buffer additional movements on top [-1..1]
+    NormalizeBuffer,
+    /// Pass the pixel distance in a normalized way and drop any overflow [-1..1]
+    NormalizeDrop,
+    /// Just pass the pixel distance travelled over to the code directly (??..??)
+    PassThrough,
```

Comment:
Help me understand the use case for this, and why a user shouldn't just use [`mouse_position`](https://docs.amethyst.rs/master/amethyst_input/struct.InputHandler.html#method.mouse_position).

As is this complicates the specification for an axis, and I'd rather not do that if this functionality is redundant.